packages/tools/dtc/package.mk

```diff
@@ -46,7 +46,15 @@ post_makeinstall_host() {
   touch $TOOLCHAIN/lib/$PKG_PYTHON_VERSION/site-packages/pylibfdt/__init__.py
 }
 
+pre_make_target() {
+  make clean
+}
+
 makeinstall_target() {
```

Comment:
I'd suggest making the new additions $PROJECT based so we're not adding them to images where they're not really relevant, e.g.

```
  makeinstall_target() {
    mkdir -p $INSTALL/usr/bin
      cp -P $PKG_BUILD/dtc $INSTALL/usr/bin

    if [ "$PROJECT" = "Amlogic" ]; then
      mkdir -p $INSTALL/usr/lib
        cp -P $PKG_BUILD/fdtput $INSTALL/usr/bin/
        cp -P $PKG_BUILD/fdtget $INSTALL/usr/bin/
        cp -P $PKG_BUILD/libfdt/libfdt.so $INSTALL/usr/lib/
    fi
```